README.md

```diff
@@ -1,14 +1,10 @@
 # netatmo-weather-adapter
 Netatmo Weather Station adapter for the [Mozilla IoT gateway](https://iot.mozilla.org).
 
-Bridges the Netatmo REST API to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
+Bridges the Netatmo REST API (via netatmo.com, not the local Netatmo device) to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
 
 ## Usage
-Normally you will just want to install this from the add-ons list provided by
-the gateway.
+In order for your gateway to read data for your Netatmo Weather Station, you need to follow the Netatmo Weather Station setup instructions provided by Netatmo. Then install and configure this from the add-ons list provided by the gateway (Settings -> Add-ons -> Netatmo Weather Station).
 
 ### Setup
-Enter the API credentials for your [Netatmo Smart Home](https://www.netatmo.com/site/connect/program#home) system in order for the adapter to read your station data.
-
-To do so, navigate to the adapter configuration on your Things Gateway dashboard. (/settings/addons/config/netatmo-weather-adapter)
-After entering your credentials, your stations should be found and can be added as new Things.
+Before attempting to add to the gateway, you need to do the normal setup of your Netatmo Weather Station (set it up on your wifi, create a Netatmo username & password, ...). Then you need to [create a Netatmo app](https://dev.netatmo.com/myaccount/createanapp). After entering and saving information for the new app, it will generate a Client id and Client secret (save these). Back on the gateway in the Netatmo Weather Station add-on (Settings -> Add-ons -> Netatmo Weather Station -> Configure), enter all the requested credentials and Apply them. On the Things screen, click + to scan for new Things and the Netatmo Weather Station sensors should now be found and can be added as new Things.
```

Comment:
I think this is getting a bit messy. I'd suggest switching to an ordered list instead and properly listing the steps.

The link to connect/program instead of createanapp was intentional. If you can't figure out what to do from that splash page you probably shouldn't be using this adapter in its current state (sorry).

The last step shouldn't instruct on how to use the gateway pairing but instead say something like "You should now see all the Netatmo Weather Stations and Healthy Home Coaches connected to the configured credentials in the device pairing screen". This is not a tutorial on how to use the gateway, but on how to set up this specific adapter.

Lastly, please try to capitalize things properly.

Reply:
FYI, I get a 404 at the existing link.